src/main/java/website/frontrow/game/Game.java

```diff
@@ -207,7 +208,8 @@ public void addToLog(String action)
     @Override
     public void levelWon()
     {
-        stop();
+        //stop();
```

Comment:
I guess I'll keep it, and add something to the backlog for automatically unpausing it.